server/src/main/java/com/lunchtime/repository/PersonRepository.java

```diff
@@ -11,4 +12,9 @@
     Person findFirstByEmail(String email);
 
     Person findPersonByPhoneNumber(String email);
+
+    @Query("select p.roleId from Person p "
+        + "where p.id in :userId "
+        + "and p.isDeleted = false")
+    long findByRoleId(long userId);
```

Comment:
If we want to find role by user id, this method's name should have `findRoleByUserId` name. Thus, this method should be in RoleRepository as we want to find Role, but nor Person (... due to this `select p.roleId` in query).

Reply:
@VozNick you should change method name or signature to make it more clear to other developers

Reply:
it won't work in RoleRepository. It 's not connected to user. I'll change the name